### DISCUSSION TITLE
Yazım hatası

### DISCUSSION THREAD

**Discussion by Abdullahoda** [IBAX-io/go-ibax#875] [Typos （English）]
### Describe the bug

Price can be understood as the money or amount to be paid to get something. Cost is the amount incurred in the production of goods, i.e. the money value of the resources involved in producing something.

4 characters the fee is

I think there is a typo here

### Reproduction

![Screenshot_20220122_163615](https://user-images.githubusercontent.com/97207588/151065898-6dddf817-416f-4aa9-8de8-04b0233a318c.jpg)

### System Info

```shell
Weaver
```

### Logs

```shell
Instead of the cost is it says the free is they should all be the same
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ibaxio** [comment]
Thanks for your attention.

In IBAX Network and other blockchains, 'cost' generally refers to the cost of invoking the contract,  means "Gas Fee", whereas in this case, 'Fee' refers to the additional cost of purchasing this particular nickname.
